package.json

```diff
@@ -60,45 +60,47 @@
     "eslint-plugin-react": "^7.5.1",
     "mofo-style": "^2.4.0",
     "optipng-bin": "^4.0.0",
-    "stylelint": "^7.13.0",
+    "stylelint": "^9.2.1",
     "stylelint-config-standard": "^16.0.0",
-    "svgo": "^1.0.3"
+    "svgo": "^1.0.3",
+    "webpack-cli": "^2.1.3"
   },
   "dependencies": {
     "autoprefixer": "^8.3.0",
-    "babel-core": "^6.26.0",
+    "babel-core": "^6.26.3",
     "babel-loader": "^7.1.4",
-    "babel-preset-es2015": "^6.18.0",
-    "babel-preset-react": "^6.16.0",
+    "babel-preset-es2015": "^6.24.1",
+    "babel-preset-react": "^6.24.1",
     "browserslist": "^3.2.6",
     "classnames": "^2.2.5",
     "document-env-vars": "^2.1.0",
     "dotenv": "^4.0.0",
     "express": "^4.16.2",
     "font-awesome": "^4.7.0",
     "helmet": "^3.9.0",
+    "lodash": "^4.17.10",
```

Comment:
what's using lodash? O_o

Reply:
this is more than likely a dependency of another dependency, and we're pinning a version that's got a security fix.